### PR TITLE
Lcg/invalid utf8

### PR DESCRIPTION
**Pull request by lamont-granquist**

### CODE REVIEW

**Comment by lamont-granquist** [comment]
@opscode/client-core @opscode/client-engineers 

this repo should probably be added to our process.

File: lib/ffi_yajl/encoder.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def encode(obj)
       yajl_gen_opts = {}
 
       yajl_gen_opts[:yajl_gen_validate_utf8] = true
+      yajl_gen_opts[:yajl_gen_validate_utf8] = @opts[:validate_utf8] == false ? false : true
```
Comment (mcquin) [inline lib/ffi_yajl/encoder.rb]: Did you mean to remove line 11, or typo?

Comment (lamont-granquist) [inline lib/ffi_yajl/encoder.rb]: lol

**Comment by danielsdeleo** [comment]
LGTM

**Pull request merged**